addons/purchase/views/stock_config_settings_views.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_config_settings_inherit_purchase" model="ir.ui.view">
```

Comment:
Follow the naming convention for view **name** and **id** .
ID should be **stock_config_settings_view_form_inherit_purchase** .